### PR TITLE
Added Isomorphic Strings

### PR DESCRIPTION
**Pull request by Mann-tech13**
### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #3236`.

### CODE REVIEW

File: src/main/java/com/thealgorithms/strings/Isomorphic.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    if(s.length() != t.length()){
+        return false;
+    }
+    // -------------------------------------------------------------
+
+    // To mark the characters of string using MAP
+    // character of first string as KEY and another as VALUE
+    // now check occurence by keeping the track with SET data structure
+    // -------------------------------------------------------------
+    boolean check = true;
```
Comment (siriak) [inline src/main/java/com/thealgorithms/strings/Isomorphic.java:20]: Please rename it to something meaningful

File: src/main/java/com/thealgorithms/strings/Isomorphic.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    Map<Character, Character> map = new HashMap<Character, Character>();
+    Set<Character> set = new HashSet<Character>();
```
Comment (siriak) [inline src/main/java/com/thealgorithms/strings/Isomorphic.java:22]: And these variables too

File: src/test/java/com/thealgorithms/strings/Isomorphic.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
+package com.thealgorithms.strings;
+
+import java.util.*;
+
+/**
+ * Isomorphic
+ */
+public class Isomorphic {
+
+  public static boolean isIsomorphic(String s, String t) {
```
Comment (siriak) [inline src/test/java/com/thealgorithms/strings/Isomorphic.java:10]: Please add tests and remove duplication

**Review by siriak — CHANGES_REQUESTED** [review]

File: src/test/java/com/thealgorithms/strings/Isomorphic.java (hunk 3 above)
Comment (Mann-tech13) [reply src/test/java/com/thealgorithms/strings/Isomorphic.java:10]: Okay, I'll do it by today

**Pull request closed without merging**